frontends/packages/livnote/desktop/state/data.svelte.ts

```
@@ -88,12 +74,14 @@ class DataState {
   }
 
   // Fetch all notes regardless of vault
-  async fetchAllNotes() {
+  async fetchAllNotes(selectedNotedId?: String) {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)